modules/Expectation.js

```diff
@@ -401,6 +401,12 @@ class Expectation {
       typeof this.actual
     )
 
+    assert(
+      isArray(keys),
+      'The "keys" argument in expect(actual).toIncludeKeys(keys) must be an array, not %s',
+      typeof keys
```

Comment:
i don't think `typeof` will be helpful here - this will report "object" for null as well. can we use `object-inspect` here instead?